apollo-router/templates/sandbox_index.html

```diff
@@ -64,6 +64,7 @@ <h1>Welcome to the Apollo Router</h1>
             },
             hideCookieToggle: false,
             endpointIsEditable: false,
+            enableConnectorsDebugging: true,
```

Comment:
should the name match the other similar flag `initialRequestQueryPlan`?

Reply:
@lennyburdette can do but would require refactoring & releasing studio code first

Reply:
Would `initialRequestConnectorsDebugging` make sense?

Reply:
yeah that's what i was thinking. it's certainly not a requirement, but once this is baked into router's sandbox html, we'll need to support it in perpetuity.